extensions/stackdriver/stackdriver.cc

```diff
@@ -120,6 +134,18 @@ void StackdriverRootContext::record(const RequestInfo &request_info,
   if (enableServerAccessLog()) {
     logger_->addLogEntry(request_info, peer_node_info);
   }
+  if (enableEdgeReporting()) {
```

Comment:
Is reporting every 10s too frequent for edge? Will this cause too much computation for the backend?

If we want to make it less frequent, there is only one timer per WASM vm right now, so we need to keep an edge reporting counter `n` to trigger edge reporting every `10 * n` seconds.

Reply:
every 10s is too much. took a stab at a configurable duration.